kubernetes-deploy.gemspec

```diff
@@ -22,9 +22,11 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
   spec.add_dependency "activesupport", ">= 4.2"
   spec.add_dependency "kubeclient", "~> 2.3"
+  spec.add_dependency "ejson", "1.0.1"
```

Comment:
It seems that `ejson` gem is only a wrapper around the tool in Go (https://github.com/Shopify/ejson/blob/master/rubygem/bin/ejson). Do we still it as a dependency?